src/bit-components.js

```diff
@@ -235,3 +236,19 @@ export const Mirror = defineComponent();
 export const ParticleEmitterTag = defineComponent({
   src: Types.ui32
 });
+export const AudioZone = defineComponent({
+  flags: Types.ui8
+});
+export const AudioListenerTag = defineComponent();
+export const AudioParams = defineComponent({
+  audioType: Types.ui32,
+  distanceModel: Types.ui32,
+  panningModel: Types.ui32,
+  rolloffFactor: Types.ui32,
+  refDistance: Types.ui32,
+  maxDistance: Types.ui32,
+  coneInnerAngle: Types.ui32,
+  coneOuterAngle: Types.ui32,
+  coneOuterGain: Types.ui32,
+  gain: Types.ui32
+});
```

Comment:
It seems like this is not being used, which makes sense because we already store this information in a map (`APP.audioOverrides`). 

At some point it might make sense to move these data into bitecs components, but it doesn't seem like a necessary step for what we're trying to do now:

```
  audios = new Map<AElement | number, PositionalAudio | Audio>();
  sourceType = new Map<AElement | number, SourceType>();
  audioOverrides = new Map<AElement | number, AudioSettings>();
  zoneOverrides = new Map<AElement | number, AudioSettings>();
  gainMultipliers = new Map<AElement | number, number>();
  supplementaryAttenuation = new Map<AElement | number, number>();
  clippingState = new Set<AElement | number>();
  mutedState = new Set<AElement | number>();
  isAudioPaused = new Set<AElement | number>();
```

Reply:
These are the same as bitecs components. We just didn't have bitecs available to us when we wrote them.

Reply:
Yep, removed.